pkg/system/phase4_configuring.go

```diff
@@ -1573,8 +1575,10 @@ func RemoveRuleFromNoobaaAdmissionWebhook(rule *admissionv1.RuleWithOperations)
 // AddRuleToNoobaaAdmissionWebhook adds the rule to the noobaa admission webhook if it
 // doesn't exists
 func AddRuleToNoobaaAdmissionWebhook(rule *admissionv1.RuleWithOperations) error {
-	wc := &admissionv1.ValidatingWebhookConfiguration{}
-	wc.SetName("admission-validation-webhook")
+	wc, err := FindNoobaaAdmissionController()
+	if err != nil {
+		return err
+	}
 
 	// Find the pre-existing admission webhook
 	if _, _, err := util.KubeGet(wc); err != nil {
```

Comment:
I think we can delete this step now that `wc` is already pre-filled with the information from the cluster?